README.md

```diff
@@ -42,6 +42,33 @@ Allows players in Creative mode to kill entities in one hit.
 - Additional notes:
   - This only works on non-player entities.
 
+### disableMobSpawningInNether
```

Comment:
Could we have these in alphabetical order please, so
- `disableMobSpawningInEnd`
- `disableMobSpawningInNether`
- `disableMobSpawningInOverworld`